packages/desktop/views/onboarding/views/complete-onboarding/views/FinishOnboardingView.svelte

```diff
@@ -22,16 +23,13 @@
     $: appName =
         $onboardingProfile?.network?.id === SupportedNetworkId.Iota ? LedgerAppName.Iota : LedgerAppName.Shimmer
 
-    function onContinueClick(): void {
-        if ($isOnboardingLedgerProfile) {
-            checkOrConnectLedger(_continue, false, appName)
-        } else {
-            void _continue()
-        }
-    }
-
-    async function _continue(): Promise<void> {
+    async function onContinueClick(): Promise<void> {
         try {
+            if ($isOnboardingLedgerProfile) {
+                await checkOrConnectLedgerAsync(appName)
+            } else {
+                await checkOrUnlockStrongholdAsync()
```

Comment:
We shouldn't need this because stronghold is already unlocked and we have not set a timer to lock it yet?

Reply:
True, I'll remove it, since the password clear interval is set in `login()`

Reply:
We have similar logic in `BalanceFinderView`, so I removed it there as well.